rpc_jobs/rpc_upgrades.yml

```diff
@@ -245,7 +245,7 @@
       - "pike_to_queens_inc"
       - "pike_to_rocky_inc"
       - "queens_to_rocky_inc"
-      - "r16.0.0-beta.1_to_queens_inc"
```

Comment:
Are we to assume that https://github.com/rcbops/rpc-gating/commit/a93bc86c3c13c1e806bbf5481998119ac76a7c1d added the wrong one and that this fixes it? There's nothing in the PR message or commit msg to explain the replacement.

Reply:
Updated my initial comment to help answer your question.